testing/e2e/bazel.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Title        bazel.sh
+# Description  Bazel utilities for setting up a WORKSPACE scoped test environment
+#================================================================================
+
+function write_workspace_file() {
+  local file_name=$1
+  local workspace_name=$2
+
+  export WIX_BUILD_TOOLS_DIR=${TEST_TMPDIR}/wix_build_tools
+  cat >"${file_name}" <<EOF
+workspace(name = "${workspace_name}")
+local_repository(
+    name = "wix_build_tools",
+    path = "${WIX_BUILD_TOOLS_DIR}",
+)
+EOF
+}
+
+function create_workspace_file_targeting_wix_build_tools() {
+  write_workspace_file "${1:-WORKSPACE}" "${2:-wix_build_tools_tests}"
+}
+
+function create_new_random_workspace() {
```

Comment:
Not sure I understand what this function creates?
WORKSPACE needs to have a WORKSPACE file :) but this method only creates random dir and adds in BUILD file

Reply:
Correct, I'll rename to `create_new_workspace_random_directory`

Reply:
Excluded the `BUILD` file creation to a separate method as well, no need to share responsibilities on a single function.